docs/openapi.yaml

```diff
@@ -975,6 +1365,190 @@ components:
+          type: string
+          example: NOK
+          enum:
+            - NOK
+        id:
+          type: string
+          example: agr_5kSeqzFAMkfBbc
+          description: Uniquely identifies this agreement
+          readOnly: true
+        interval:
+          type: string
+          example: WEEK
+          description: Interval for subscription
+          enum:
+            - DAY
+            - WEEK
+            - MONTH
+            - YEAR
+        intervalCount:
+          type: integer
+          format: int32
+          example: 2
+          description: 'Number of intervals between charges. Example: interval=WEEK, intervalCount=2 to be able to charge every two weeks. Charges should occur at least once a year.'
+          minimum: 1
+          maximum: 31
+          exclusiveMinimum: false
+          exclusiveMaximum: false
+        price:
+          type: integer
+          format: int32
+          example: 7900
+          description: Treated as a whole sum postfixed with two decimals for cents, eg 234 = 2.34 NOK
+        productDescription:
+          type: string
+          example: Fin abonnementspakke
+        productName:
+          type: string
+          example: Pluss-abonnement
+          description: Name of the product being subscribed to.
+        start:
+          type: string
+          example: '2019-01-01T00:00:00Z'
+          description: Date when agreement was started.
+          readOnly: true
+        status:
+          type: string
+          example: ACTIVE
+          description: Status of the agreement.
+          enum:
+            - PENDING
+            - ACTIVE
+            - STOPPED
+            - EXPIRED
+        stop:
+          type: string
+          example: '2019-01-01T00:00:00Z'
+          description: Date when agreement was stopped.
+          readOnly: true
+        merchantAgreementUrl:
+          type: string
+          example: https://www.merchant.no/subscriptions/1234/
+          description: URL where Vipps can redirect the customer to view/administer their subscription.
+        sub:
+          type: string
+          example: 8d7de74e-0243-11eb-adc1-0242ac120002
+          description: User identifier (subject). Will be null if profile data was not requested.
+        userinfoUrl:
+          type: string
+          example: https://api.vipps.no/vipps-userinfo-api/userinfo/8d7de74e-0243-11eb-adc1-0242ac120002
+          description: The full path of the URL for the userinfo endpoint where the user data can be retrieved. Will be null if profile data was not requested.
+        variableAmount:
+                  $ref: "#/components/schemas/VariableAmount"
+      title: Agreement
+
+    PatchAgreementV3:
+      type: object
+      properties:
+        suggestedMaxAmount:
+          type: integer
+          format: int32
+          example: 7900
+          minimum: 100
+          description: Treated as a whole sum postfixed with two decimals for cents, eg 234 = 2.34 NOK
```

Comment:
From other docs: `Amounts are specified in minor units. For Norwegian kroner (NOK) that means 1 kr = 100 øre. Example: 499 kr = 49900 øre.`

See eCom: https://github.com/vippsas/vipps-ecom-api/blob/553a601a997435f2915e7c935de9f297b29f229b/docs/swagger.yaml#L1489